docs/middleware.md

```diff
@@ -50,7 +60,7 @@ export type Middleware = (
 
 The function can be a default export and as such, does **not** have to be named `middleware`. Though this is a convention. Also note that you only need to make the function `async` if you are running asynchronous code.
 
-**Note:** Edge Functions are currently in Beta. The API might change as we look to continually make improvements.
+[Read the full Middleware API reference.](/docs/api-reference/next/server.md)
```

Comment:
Since the version table is usually collapsed, we might want to keep the “beta” label here too

Reply:
I'd say let's keep it open, then (I prefer listing Beta up at the top) 👍